### PR TITLE
fix for login page ready

### PR DESCRIPTION
**Pull request by sergioloporto**
https://github.com/globaldothealth/list/issues/2297

### CODE REVIEW

File: verification/curator-service/ui/src/components/SnackbarAlert/index.tsx [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+import React, { ReactNode } from 'react';
 import Snackbar from '@material-ui/core/Snackbar';
 import MuiAlert, { AlertProps } from '@material-ui/lab/Alert';
 
 interface SnackbarAlertProps {
-    isOpen: boolean;
+    isOpen: boolean | undefined;
     onClose: (value: boolean) => void;
-    message: string;
+    message: string | ReactNode;
```
Comment (maciekz1996) [inline verification/curator-service/ui/src/components/SnackbarAlert/index.tsx:8]: What was the reason for this change?

**Review by maciekz1996 — CHANGES_REQUESTED** [review]
It looks way better on mobiles than before, good job 👍 While testing on iPhone X I saw that some text still spills out of the page, I think it would be best to put this "More information" section under first text block. Also 3 last images at the bottom of the page should be centered horizontally the same as images above. 

Screen Shot 2022-02-04 at 13 25 44
Screen Shot 2022-02-04 at 13 25 30

File: verification/curator-service/ui/src/components/SnackbarAlert/index.tsx (hunk 1 above)
Comment (sergioloporto) [reply verification/curator-service/ui/src/components/SnackbarAlert/index.tsx:8]: it is necessary to circumvent this error:
image

**Comment by sergioloporto** [comment]
Ready for MAP.

Now working on popup for Data Site

**Review by maciekz1996 — APPROVED** [review]

**Review by abhidg — APPROVED** [review]

**Comment by iamleeg** [comment]
Just did the rebase with main: if CI passes then I'll merge.

**Pull request merged**